helix-core/src/test/java/org/apache/helix/common/ZkTestBase.java

```diff
@@ -107,12 +109,23 @@
   private Map<String, Map<String, HelixZkClient>> _liveInstanceOwners = new HashMap<>();
 
   public static final String ZK_ADDR = "localhost:2183";
+  private static final String ZK_PREFIX = "localhost:";
+  private static final int ZK_START_PORT = 2183;
   protected static final String CLUSTER_PREFIX = "CLUSTER";
   protected static final String CONTROLLER_CLUSTER_PREFIX = "CONTROLLER_CLUSTER";
   protected final String CONTROLLER_PREFIX = "controller";
   protected final String PARTICIPANT_PREFIX = "localhost";
   private static final long MANUAL_GC_PAUSE = 4000L;
 
+  /*
+   * Multiple ZK references
+   */
+  protected int _numZk = 0;
```

Comment:
Instead of multiple lists, can we have a class like a tuple in Python to hold the related objects? That would be much easier to use comparing if I need to remember the index in every test case.

Reply:
Discussed offline.

Making this 

>   protected Map<String, ZkServer> _zkServerMap = new HashMap<>();
  protected Map<String, HelixZkClient> _helixZkClientMap = new HashMap<>();
  protected Map<String, ClusterSetup> _clusterSetupMap = new HashMap<>();
  protected Map<String, BaseDataAccessor> _baseDataAccessorMap = new HashMap<>();
for the time being.